server/sonar-server/src/main/java/org/sonar/server/authentication/CookieUtils.java

```diff
@@ -40,4 +44,20 @@ private CookieUtils() {
       .filter(cookie -> cookieName.equals(cookie.getName()))
       .findFirst();
   }
+
+  public static Cookie createCookie(String name, @Nullable String value, boolean httpOnly, int expiry, HttpServletRequest request) {
+    Cookie cookie = new Cookie(name, value);
+    // Path is set "/" in order to allow rails to be able to remove cookies
+    // TODO When logout when be implemented in Java (SONAR-7774), following line should be replaced by
```

Comment:
![INFO](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-info.png) Complete the task associated to this TODO comment. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1135)